web/viewer.css

```diff
@@ -1509,7 +1509,8 @@ html[dir='rtl'] #documentPropertiesOverlay .row > * {
   display: none;
 }
 
-@media screen and (min-resolution: 2dppx) {
+@media screen and (min-resolution: 2dppx),
+       screen and (-webkit-min-device-pixel-ratio: 2) {
```

Comment:
This will unfortunately break horribly during the CSS preprocessing stage, leading to the Firefox addon/built-in version not working correctly!
You probably need to do something like this instead:
```
@media screen and (-webkit-min-device-pixel-ratio: 2),
@media screen and (min-resolution: 2dppx) {
```